.github/workflows/pr_validate.yml

```diff
@@ -11,10 +11,14 @@ jobs:
         image: fauna/faunadb:latest
         ports:
           - 8443:8443
+        env:
+          FLAG_ACCOUNT_CHANGE_FEEDS: "true"
```

Comment:
These flags aren't needed anymore. They're enabled by default on the Docker image.